resolver/addressable_resolver.go

```diff
@@ -153,6 +159,14 @@ func (r *URIResolver) URIFromObjectReference(ctx context.Context, ref *corev1.Ob
 		return nil, apierrs.NewBadRequest("ref is nil")
 	}
 
+	// try custom resolvers first
+	for _, resolver := range r.resolvers {
+		handled, url, err := resolver(ctx, ref)
```

Comment:
we likely want to check the error here before handled. Looks like err is lots when handled is false.

I am not sure what is right, but it could do with a comment to say if we can ignore err if handled is false here.

Reply:
added a comment. Please TAL.